app/assets/stylesheets/active_admin/mixins/_variables.scss

```diff
@@ -2,33 +2,233 @@
+$badge-background-color: rgba(0, 0, 0, 0.35) !default;
+
+$badge-border-radius: 10px !default;
+$badge-font-weight: $font-weight-normal !default;
+$badge-line-height: 1 !default;
+
+//== Breadcrumbs
+
+$breadcrumb-color: #7f8c8d !default;
+$breadcrumb-separator-color: #7f8c8d !default;
+$breadcrumb-separator: "/" !default;
+
+//== Flash Messages
+
+$flash-message-font-weight: $font-weight-bold !default;
+$flash-message-padding: 12px $horizontal-page-margin !default;
+
+$flash-message-default-background-color: $warning-color !default;
+$flash-message-default-text-color: #fff !default;
+
+$flash-message-notice-background-color: $success-color !default;
+$flash-message-notice-text-color: #fff !default;
+
+$flash-message-error-background-color: $error-color !default;
+$flash-message-error-text-color: #fff !default;
+
+//== Modals
+
+$modal-content-padding: 15px !default;
+$modal-heading-padding: 10px 15px !default;
+
+$modal-background-color: #fff !default;
+$modal-backdrop-background-color: rgba(0, 0, 0, 0.5) !default;
+$modal-heading-border-color: #eee !default;
+
+//== Navbar
+
+$navbar-border-radius: $border-radius-base !default;
+$navbar-padding-horizontal: 10px !default;
+$navbar-padding-vertical: 10px !default;
+
+$navbar-color: #fff !default;
+$navbar-background-color: $primary-color !default;
+
+$navbar-link-color: #fff !default;
+$navbar-link-hover-color: #fff !default;
+$navbar-link-hover-background-color: lighten($primary-color, 8%) !default;
+$navbar-link-active-color: #fff !default;
+$navbar-link-active-background-color: lighten($primary-color, 8%) !default;
+
+//== Panels
+
+$panel-background-color: lighten(#ecf0f1, 4%) !default;
+$panel-body-padding: 15px !default;
+$panel-heading-background-color: #ecf0f1 !default;
+$panel-heading-padding: 10px 15px !default;
+$panel-border-radius: $border-radius-base !default;
+
+//== Status Tags
+
+$status-tag-border-radius: $border-radius-base !default;
+$status-tag-padding: 0.2em 0.5em !default;
+
+$status-tag-color: #fff !default;
+$status-tag-no-background-color: #7f8c8d !default;
+$status-tag-yes-background-color: #27ae60 !default;
+
+//== Tabs
+
+$tabs-border-color: #d3d3d3 !default;
+
+$tabs-link-hover-background-color: darken(#fff, 10%) !default;
+
+$tabs-active-link-hover-background-color: #fff !default;
+$tabs-active-link-hover-border-color: $tabs-border-color !default;
+$tabs-active-link-hover-color: $text-color !default;
+
+//== Title Bar
+
+$title-bar-background-color: lighten(#ecf0f1, 2%) !default;
+$title-bar-color: $text-color !default;
+
+$title-bar-heading-font-size: 1.9em !default;
+
+//== Date Picker
+
+$date-picker-header-background-color: $primary-color;
+$date-picker-header-link-color: #fff !default;
+$date-picker-header-link-hover-color: #fff !default;
+
+$date-picker-calendar-background-color: #fff !default;
+$date-picker-calendar-link-hover-background-color: #ecf0f1 !default;
+$date-picker-calendar-link-active-background-color: darken(#ecf0f1, 10%) !default;
```

Comment:
Color literals like `#ecf0f1` should only be used in variable declarations; they should be referred to via variable everywhere else.